config/examples/Creality/Ender-3 Pro/CrealityV422/Configuration_adv.h

```diff
@@ -1278,7 +1278,7 @@
  * vibration and surface artifacts. The algorithm adapts to provide the best possible step smoothing at the
  * lowest stepping frequencies.
  */
-#define ADAPTIVE_STEP_SMOOTHING
+//#define ADAPTIVE_STEP_SMOOTHING
```

Comment:
I've done months of printing with this enabled on a V422 board with no issues and likely the non-silent version of V422 could benefit from leaving it enabled.

Reply:
This was also commented out in CrealityV427. I was just looking to make the two consistent.